modules/serving-HA-components.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-config-HA.adoc
+
+[id="serving-HA-components_{context}"]
+= Knative Serving high availability components
+
+HA is available by default for the `autoscaler-hpa`, `controller`, `activator` , `kourier-control`, and `kourier-gateway` components.
+
+[id="serving-disabling-HA_{context}"]
+== Disabling high availability
+
+You can disable HA for these components by changing the configuration of the `KnativeServing.spec.highAvailability` field.
+
+[IMPORTANT]
+====
+Do not modify any YAML contained inside the `config` field. Some of the configuration values in this field are injected by the {ServerlessOperatorName}, and modifying them will cause your deployment to become unsupported.
+====
+
+.Prerequisites
+* An {product-title} account with cluster administrator access.
+* Installed the {ServerlessOperatorName} and Knative Serving.
+
+.Procedure
+
+. In the {product-title} web console, navigate to the *Catalog* → *OperatorHub* → *Installed Operators* page.
```

Comment:
It might all be the same in the end, but suggest using `->` instead of the character you use for arrows. It's what we say to use in our guidelines, and good to follow just in case there's any difference in appearance.

Reply:
Didn't even realise this was in the guidelines, that's so much better, thanks!